test/utils/do-factory.ts

```diff
@@ -156,6 +155,32 @@ export class DoFactory {
         );
     }
 
+    //Remove this when PersonenkontextDo is remove dor set to deprecated
+    public static createPersonenkontextDo<WasPersisted extends boolean>(
+        this: void,
+        withId: WasPersisted,
+        params: Partial<PersonenkontextDo<boolean>> = {},
+    ): PersonenkontextDo<WasPersisted> {
```

Comment:
Set it to deprecated, otherwise it will be forgotten. Or can it be deleted already?